dotcom-rendering/src/lib/contributions.test.ts

```diff
@@ -1,12 +1,6 @@
 import { setCookie } from '@guardian/libs';
-import {
-	getLastOneOffContributionTimestamp,
-	isRecentOneOffContributor,
-	lazyFetchEmailWithTimeout,
-	ONE_OFF_CONTRIBUTION_DATE_COOKIE,
-	SUPPORT_ONE_OFF_CONTRIBUTION_COOKIE,
-} from './contributions';
-import { getIdApiUserData } from './getIdapiUserData';
+import type { PurchaseInfo } from '@guardian/support-dotcom-components/dist/shared/src/types/targeting';
```

Comment:
Should we update the package to export this type? @tjmw

Reply:
It can actually be inferred as is done in https://github.com/guardian/dotcom-rendering/blob/1f1324951172d79c5e29ceabda1a3eece589c4d6/dotcom-rendering/src/lib/contributions.ts#L276